docs/standard/language-independence-and-language-independent-components.md

```diff
@@ -172,6 +172,7 @@ The .NET [common type system](base-types/common-type-system.md) includes a numbe
 |<xref:System.Int16>|16-bit signed integer|
 |<xref:System.Int32>|32-bit signed integer|
 |<xref:System.Int64>|64-bit signed integer|
+|<xref:System.Half>|Half-precision floating-point value|
```

Comment:
Same note here as before

Reply:
The formal IEEE 754 name is `binary16` for `half`, `binary32` for single, and `binary64` for `double`.

Reply:
@tannergooding we should probably update the API docs, as it says float16: https://docs.microsoft.com/en-us/dotnet/api/system.half?view=net-5.0